o4/o4_prune.py

```diff
@@ -0,0 +1,52 @@
+
+from pathlib import Path
+from time import time
+
+from o4_fstat import prune_fstat_cache
+
+
+def prune_data(o4_dir):
+    prune_fstat_cache(o4_dir)
+    prune_cleaned(o4_dir)
+
+
+def prune_cleaned(o4_dir):
+    '''
+    Remove all read-only files from an abandoned cleaned directory
+    (unless it's too recent).
+    '''
+    cleaned = Path(o4_dir) / 'cleaned'
+    if not cleaned.exists():
+        return
+    if not cleaned.is_dir():
+        print('*** WARNING: "cleaned" is not a directory! Not trying to prune it.')
+        return
+    age = time() - cleaned.stat().st_mtime
+    if age < 86400 * 2:
+        print('*** INFO: You have files left behind by a recent "clean" operation.\n'
+              '          It will be pruned soon.'
+             )
```

Comment:
come on yapf.
```suggestion
        print('*** INFO: You have files left behind by a recent "clean" operation.',
              '          It will be pruned soon.', sep='\n')
```